CurrentTimeApp.Components/CurrentTimeApp.Components.csproj

```diff
@@ -15,7 +15,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="7.0.1" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="7.0.2" />
```

Comment:
This change has been included in https://github.com/jjliggett/CurrentTimeApp/pull/96 which updates the dependencies and the lock file.